diff --git i/src/PixiEditor.AnimationRenderer.FFmpeg/FFMpegRenderer.cs w/src/PixiEditor.AnimationRenderer.FFmpeg/FFMpegRenderer.cs
index 73e24b8..cde9aa3 100644
--- i/src/PixiEditor.AnimationRenderer.FFmpeg/FFMpegRenderer.cs
+++ w/src/PixiEditor.AnimationRenderer.FFmpeg/FFMpegRenderer.cs
@@ -22,9 +22,7 @@ public class FFMpegRenderer : IAnimationRenderer
     public async Task<bool> RenderAsync(List<Image> rawFrames, string outputPath, CancellationToken cancellationToken,
         Action<double>? progressCallback = null)
     {
-        string path = $"ThirdParty/{IOperatingSystem.Current.Name}/ffmpeg";
-
-        string binaryPath = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), path);
+        string binaryPath = "@binaryPath@";
 
         GlobalFFOptions.Configure(new FFOptions() { BinaryFolder = binaryPath });
 
@@ -87,9 +85,7 @@ public class FFMpegRenderer : IAnimationRenderer
     public bool Render(List<Image> rawFrames, string outputPath, CancellationToken cancellationToken,
         Action<double>? progressCallback)
     {
-        string path = $"ThirdParty/{IOperatingSystem.Current.Name}/ffmpeg";
-
-        string binaryPath = Path.Combine(Path.GetDirectoryName(Environment.ProcessPath), path);
+        string binaryPath = "@binaryPath@";
 
         GlobalFFOptions.Configure(new FFOptions() { BinaryFolder = binaryPath });
 

