diff --git a/platform/linuxbsd/detect.py b/platform/linuxbsd/detect.py
index 36644d5..b95a18f 100644
--- a/platform/linuxbsd/detect.py
+++ b/platform/linuxbsd/detect.py
@@ -202,6 +202,10 @@ def configure(env):
         env.ParseConfig("pkg-config xrender --cflags --libs")
         env.ParseConfig("pkg-config xi --cflags --libs")
 
+        env.ParseConfig("pkg-config xfixes --cflags --libs")
+        env.ParseConfig("pkg-config glu --cflags --libs")
+        env.ParseConfig("pkg-config zlib --cflags --libs")
+
     if env["touch"]:
         env.Append(CPPDEFINES=["TOUCH_ENABLED"])
 
@@ -301,7 +305,7 @@ def configure(env):
     if os.system("pkg-config --exists alsa") == 0:  # 0 means found
         env["alsa"] = True
         env.Append(CPPDEFINES=["ALSA_ENABLED", "ALSAMIDI_ENABLED"])
-        env.ParseConfig("pkg-config alsa --cflags")  # Only cflags, we dlopen the library.
+        env.ParseConfig("pkg-config alsa --cflags --libs")
     else:
         print("Warning: ALSA libraries not found. Disabling the ALSA audio driver.")
 
@@ -334,7 +338,7 @@ def configure(env):
         if env["udev"]:
             if os.system("pkg-config --exists libudev") == 0:  # 0 means found
                 env.Append(CPPDEFINES=["UDEV_ENABLED"])
-                env.ParseConfig("pkg-config libudev --cflags")  # Only cflags, we dlopen the library.
+                env.ParseConfig("pkg-config libudev --cflags --libs")
             else:
                 env["udev"] = False
                 print("Warning: libudev development libraries not found. Disabling controller hotplugging support.")
