diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6104c17..4ab176d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,9 +11,9 @@ if(WIN32)
 set(VCPKG_TARGET_TRIPLET "x64-windows-static" CACHE STRING "")
 endif()
 
-set(VCPKG_OVERLAY_PORTS "${CMAKE_CURRENT_LIST_DIR}/dependencies/vcpkg_overlay_ports")
-set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake"
-  CACHE STRING "Vcpkg toolchain file")
+# set(VCPKG_OVERLAY_PORTS "${CMAKE_CURRENT_LIST_DIR}/dependencies/vcpkg_overlay_ports")
+# set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake"
+#   CACHE STRING "Vcpkg toolchain file")
 
 project(Cemu VERSION 0.1)
 set(CMAKE_CXX_STANDARD 20)
@@ -55,18 +55,15 @@ if(WIN32)
 option(ENABLE_DIRECTAUDIO "Enables the directaudio backend" ON)
 option(ENABLE_XAUDIO "Enables the xaudio backend" ON)
 endif()
-option(ENABLE_CUBEB "Enabled cubeb backend" ON)
+option(ENABLE_CUBEB "Enabled cubeb backend" OFF)
 
 option(ENABLE_WXWIDGETS "Build with wxWidgets UI (Currently required)" ON)
 
 find_package(SDL2 CONFIG REQUIRED)
-find_package(CURL CONFIG REQUIRED)
 find_package(pugixml CONFIG REQUIRED)
-find_package(imgui CONFIG REQUIRED)
 find_package(RapidJSON CONFIG REQUIRED)
 find_package(Boost COMPONENTS program_options filesystem nowide REQUIRED)
 find_package(libzip REQUIRED)
-find_package(glslang REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(zstd CONFIG REQUIRED)
 
@@ -84,7 +81,8 @@ if(ENABLE_DISCORD_RPC)
 endif()
 
 if(ENABLE_WXWIDGETS)
-    find_package(wxWidgets CONFIG REQUIRED)
+    find_package(wxWidgets REQUIRED COMPONENTS core base )
+    include(${wxWidgets_USE_FILE})
 endif()
 
 find_package(OpenSSL REQUIRED)
@@ -93,10 +91,9 @@ find_package(X11)
 # find a better way to handle this
 link_libraries(${Boost_LIBRARIES})
 link_libraries(${X11_LIBRARIES})
-link_libraries(SDL2::SDL2 SDL2::SDL2main SDL2::SDL2-static)
-if(ENABLE_WXWIDGETS)
-link_libraries(wx::core wx::base)
-endif()
+# if(ENABLE_WXWIDGETS)
+# link_libraries(wx::core wx::base)
+# endif()
 
 if(ENABLE_CUBEB)
     option(BUILD_TESTS "" OFF)
@@ -110,7 +107,5 @@ if(ENABLE_CUBEB)
 endif()
 
 add_subdirectory(dependencies/ih264d)
-add_subdirectory(dependencies/ZArchive)
-
 add_subdirectory(src)
 
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 005cfa2..e25bc61 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -80,15 +80,22 @@ set_target_properties(CemuBin PROPERTIES
         OUTPUT_NAME "Cemu"
         )
 
+find_package(PkgConfig)
+find_package( CURL REQUIRED )
+# pkg_check_modules(imgui imgui REQUIRED)
+# pkg_check_modules(glslang  glslang REQUIRED)
+
 target_link_libraries(CemuBin PRIVATE CemuCommon CemuComponents CemuCafe CemuConfig CemuGui CemuAudio CemuInput CemuUtil)
 target_link_libraries(CemuBin PRIVATE CemuAsm)
 target_link_libraries(CemuBin PRIVATE OpenSSL::SSL)
 target_link_libraries(CemuBin PRIVATE ZLIB::ZLIB)
 target_link_libraries(CemuBin PRIVATE ${wxWidgets_LIBRARIES})
-target_link_libraries(CemuBin PRIVATE CURL::libcurl)
-target_link_libraries(CemuBin PRIVATE imgui::imgui)
+target_link_libraries(CemuBin PRIVATE CURL)
+target_link_libraries(CemuBin PRIVATE SDL2)
+# target_link_libraries(CemuBin PRIVATE imgui::imgui)
 target_link_libraries(CemuBin PRIVATE pugixml pugixml::static pugixml::pugixml)
-
+# target_link_libraries(CemuBin PRIVATE wxWidgets::base wxWidgets::core)
+target_link_libraries(CemuBin PRIVATE ${wxWidgets_LIBRARIES})
 if(ENABLE_CEMUHOOK)
 target_link_libraries(CemuBin PRIVATE CemuCemuhook)
 endif()
@@ -99,3 +106,17 @@ CemuCommon CemuAudio CemuInput CemuComponents CemuCafe CemuConfig CemuGui imguiI
 # needed because of some cyclic dependencies. fix this
 target_link_libraries(CemuBin PUBLIC
 CemuCommon CemuInput CemuComponents CemuCafe CemuResource CemuGui CemuAsm)
+
+
+# include_directories(
+#     SYSTEM 
+#     ${CURL_INCLUDE_DIRS} 
+#     ${IMGUI_INCLUDE_DIRS}
+#     ${glslang_INCLUDE_DIRS}
+# )
+# target_link_libraries(CemuBin
+#   ${CURL_LIBRARIES}
+#   ${IMGUI_LIBRARIES}
+#   ${glslang_LIBRARIES}
+# )
+#
diff --git a/src/main.cpp b/src/main.cpp
index 98f829d..542dd92 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -33,7 +33,7 @@
 #endif
 
 #define SDL_MAIN_HANDLED
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 #if BOOST_OS_LINUX > 0
 #define _putenv(__s) putenv((char*)(__s))
