diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6104c17..92724d8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,9 +11,9 @@ if(WIN32)
 set(VCPKG_TARGET_TRIPLET "x64-windows-static" CACHE STRING "")
 endif()
 
-set(VCPKG_OVERLAY_PORTS "${CMAKE_CURRENT_LIST_DIR}/dependencies/vcpkg_overlay_ports")
-set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake"
-  CACHE STRING "Vcpkg toolchain file")
+# set(VCPKG_OVERLAY_PORTS "${CMAKE_CURRENT_LIST_DIR}/dependencies/vcpkg_overlay_ports")
+# set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake"
+#   CACHE STRING "Vcpkg toolchain file")
 
 project(Cemu VERSION 0.1)
 set(CMAKE_CXX_STANDARD 20)
@@ -60,13 +60,10 @@ option(ENABLE_CUBEB "Enabled cubeb backend" ON)
 option(ENABLE_WXWIDGETS "Build with wxWidgets UI (Currently required)" ON)
 
 find_package(SDL2 CONFIG REQUIRED)
-find_package(CURL CONFIG REQUIRED)
 find_package(pugixml CONFIG REQUIRED)
-find_package(imgui CONFIG REQUIRED)
 find_package(RapidJSON CONFIG REQUIRED)
 find_package(Boost COMPONENTS program_options filesystem nowide REQUIRED)
 find_package(libzip REQUIRED)
-find_package(glslang REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(zstd CONFIG REQUIRED)
 
@@ -84,7 +81,8 @@ if(ENABLE_DISCORD_RPC)
 endif()
 
 if(ENABLE_WXWIDGETS)
-    find_package(wxWidgets CONFIG REQUIRED)
+    # find_package(wxWidgets CONFIG REQUIRED)
+    find_package(wxWidgets REQUIRED COMPONENTS core base )
 endif()
 
 find_package(OpenSSL REQUIRED)
@@ -114,3 +112,20 @@ add_subdirectory(dependencies/ZArchive)
 
 add_subdirectory(src)
 
+find_package(PkgConfig)
+pkg_check_modules(CURL libcurl REQUIRED)
+pkg_check_modules(IMGUI libcurl REQUIRED)
+pkg_check_modules(glslang libcurl REQUIRED)
+
+include_directories(
+    SYSTEM 
+    ${CURL_INCLUDE_DIRS} 
+    ${IMGUI_INCLUDE_DIRS}
+    ${glslang_INCLUDE_DIRS}
+)
+target_link_libraries(CemuBin
+  ${CURL_LIBRARIES}
+  ${IMGUI_LIBRARIES}
+  ${glslang_LIBRARIES}
+)
+
